src/variables.py

```diff
@@ -44,30 +44,31 @@
     MELLOW_CONTRACT_ADDRESS = Web3.to_checksum_address(MELLOW_CONTRACT_ADDRESS)
 VAULT_DIRECT_DEPOSIT_THRESHOLD = Web3.to_wei(*os.getenv('VAULT_DIRECT_DEPOSIT_THRESHOLD', '1 ether').split(' '))
 
-# rabbit / kafka / rabbit,kafka
+# rabbit
 MESSAGE_TRANSPORTS = os.getenv('MESSAGE_TRANSPORTS', '').split(',')
 
-# Kafka secrets
-KAFKA_BROKER_ADDRESS_1 = os.getenv('KAFKA_BROKER_ADDRESS_1')
-KAFKA_USERNAME = os.getenv('KAFKA_USERNAME')
-KAFKA_PASSWORD = os.getenv('KAFKA_PASSWORD')
-KAFKA_NETWORK = os.getenv('KAFKA_NETWORK', 'mainnet')  # or goerli
-KAFKA_TOPIC = os.getenv('KAFKA_TOPIC')
-KAFKA_GROUP_PREFIX = os.getenv('KAFKA_GROUP_PREFIX', '')
-
 # rabbit secrets
 RABBIT_MQ_URL = os.getenv('RABBIT_MQ_URL', 'ws://127.0.0.1:15674/ws')
 RABBIT_MQ_USERNAME = os.getenv('RABBIT_MQ_USERNAME', 'guest')
 RABBIT_MQ_PASSWORD = os.getenv('RABBIT_MQ_PASSWORD', 'guest')
 
+# data bus
+# gnosis nodes
+ONCHAIN_TRANSPORT_RPC_ENDPOINTS = os.getenv('ONCHAIN_TRANSPORT_RPC_ENDPOINTS', '').split(',')
+
+ONCHAIN_TRANSPORT_ADDRESS = os.getenv('ONCHAIN_TRANSPORT_ADDRESS', None)
+if ONCHAIN_TRANSPORT_ADDRESS:
+    # bot will throw exception if there is unexpected str and it's ok
```

Comment:
Better describe what to do to resolve exception, smth like
```
Expecting onchain databus contract address 
```

Reply:
Done